sweep/aggregator.go

```diff
@@ -202,7 +202,7 @@ func (b *BudgetAggregator) filterInputs(inputs InputsMap) InputsMap {
 		op := pi.OutPoint()
 
 		// Get the size and skip if there's an error.
-		size, _, err := pi.WitnessType().SizeUpperBound()
+		vb, _, err := pi.WitnessType().SizeUpperBound()
```

Comment:
Umm this is very suspicious to me that SizeUpperBound returns VBytes. Witnesses are discounted so the physical size of the witness is the number of weight units. I think if this returns the Vsize we should rename the interface function to be `VSizeUpperBound()`

Reply:
yeah turns out this is just the witness data, what's missing is the other non-witness fields in the input, which were not taking into account previously.